varinfo/cmr_search.py

```diff
@@ -15,44 +15,57 @@
                                 DirectoryCreationException)
 
 
+CmrEnvType = Literal[CMR_OPS, CMR_UAT, CMR_SIT]
+
+
 def get_granules(concept_id: str = None,
-                 shortname: str = None,
+                 collection_shortname: str = None,
                  collection_version: str = None,
                  provider: str = None,
-                 cmr_env: CMR_OPS = CMR_OPS,
-                 token: str = None) -> list:
-    ''' Search CMR given:
-        * concept_id: a collection or granule's IDs
-        * shortname/short_name: 'SNDRSNIML2CCPRET' or 'M2T1NXSLV'
+                 cmr_env: CmrEnvType = CMR_OPS,
+                 auth_header: str = None) -> list:
```

Comment:
This is really the important change in this function signature (`token` to `auth_header`). It makes this function agnostic to EDL bearer tokens vs LaunchPad, vs whatever else comes in the future.